### ISSUE TITLE
Can We Get a "Remove Cheer Comment Banner" Option?

### ISSUE THREAD

**Issue by ehbowe** [night/BetterTTV#1746]
Edit 2: Well is was working at least. The "Hide Bits" option isn't hiding the banner anymore.

Edit: It looks like the "Hide Bits" option gets rid of it. Ideally there would be a separate option for just the banner, but the Hide Bits works.

Twitch recently introduced a feature where the most recent (?) cheer comment stays at the top of chat indefinitely. There's no way to dismiss this banner, it just stays there at the top of chat. I'd love a way to hide or get rid of it through BTTV.

**Comment by simoniz0r** [comment]
I just came here to post the same thing.  I rather dislike the new pinned cheer system, but I do not want to hide cheers completely.

Thanks for the great work on BTTV.

**Comment by thinkpad4** [comment]
I second this request

**Comment by night** [comment]
Unfortunately we're not willing to add yet another option to our already cluttered settings. They will have to remain grouped. Blame Twitch for making nonsensical trash like pinned bits instead of useful tech like pinned messages for announcements.

**Comment by ehbowe** [comment]
It's fine if they remain grouped, but right now even the "Hide Bits" option
doesn't get rid of the cheer banner.

**Comment by night** [comment]
It definitely does unless you can provide evidence otherwise..

**Comment by ehbowe** [comment]
Not sure this counts as evidence, but I tried it on two different channels.
It worked on one and not the other. On the one it worked on, I could just
toggle it on and off and the banner would appear/reappear as you would
expect.

However on the other channel the banner stayed no matter what. I tried
reloading the page several times as well and it didn't work. I can try to
screen cap it the next time that streamer is on, but I don't think they
stream on weekends. I only tested it on two streams.

FWIW, the stream it worked on was GabySpartz and the one it didn't was
NumotTheNummy. One other detail is that I'm subscribed to Numot via twitch
prime but I'm not subscribed to Gaby at all.
